File: pset6/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int
# get_height
while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

for i in range(height):
    # spaces
    print((height - 1 - i) * " ", end="")
    # hashes
    print((i + 1) * "#")

File: pset1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("height: ");
    }
    while (h < 1 || h > 8);

    for (int r = 0; r < h; r++)
    {
//spaces
        for (int s = 0; s < h - r - 1; s++)
        {
            printf(" ");
        }
//hashes
        for (int c = 0; c <= r; c++)
        {
            printf("#");
        }
//lines
        printf("\n");
    }
}